### PR TITLE
Update DisableAutoConfig Policy

### PR DESCRIPTION
**Pull request by persident**
Update strings and description to align with latest OneDrive release (22.151.0717.0001 - Production ring)
see OneDrive.admx and OneDrive.adml
old policy name = Prevent authentication from automatically happening
new policy name = Disable silently sign in users to the OneDrive sync app with an existing credential that is made available to Microsoft applications

### CODE REVIEW

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by HansBrender** [comment]
so the first firsion I have seen  "Disable silently sign in users to the OneDrive sync app with an existing credential that is made available to Microsoft applications"  was 22.116  (Insider). In the production this GPO comes with 22.121 
But on the docs it is not updated.

**Comment by MikePlumleyMSFT** [comment]
@gacarini can you review?

**Comment by gacarini** [comment]
@cartergreen to review since he is driving this feature.

File: OneDrive/use-group-policy.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ The OneDrive Group Policy objects work by setting registry keys on the computers
 
 - (DehydrateSyncedTeamSites) [Convert synced team site files to online-only files](use-group-policy.md#convert-synced-team-site-files-to-online-only-files)
 
-- (DisableAutoConfig) [Prevent authentication from automatically happening](use-group-policy.md#prevent-authentication-from-automatically-happening)
+- (DisableAutoConfig) [Disable silently sign in users to the OneDrive sync app with an existing credential that is made available to Microsoft applications](use-group-policy.md#disable-silently-sign-in-users-to-the-onedrive-sync-app-with-an-existing-credential-that-is-made-available-to-microsoft-applications)
```
Comment (cartergreen) [inline OneDrive/use-group-policy.md:86]: I'd prefer to keep the title as is but I do like the change to the description of the policy.  Can you please revert the title change and then I will approve it. @persident

**Comment by MikePlumleyMSFT** [comment]
Closing out old PR due to merge conflicts. If this article still needs updates, please open a new PR.

**Pull request closed without merging**